server/views/index.handlebars

```diff
@@ -54,11 +53,10 @@
     <div class='button-wrapper'>
 
       <div class='col-sm-4'>
-        <a href='/public/downloads/Zapsnap-1.0.1.dmg' class='button'>
+        <a href='/public/downloads/Zapsnap-1.0.0.dmg' class='button'>
```

Comment:
nope, `1.0.1` is a newer version, we should link to the newest version from the download button